tools/modules/EnsEMBL/Web/Component/Tools/VR/InputForm.pm

```diff
@@ -0,0 +1,280 @@
+      {
+        'type'          => 'noedit',
+        'value'         => '<b>Either paste data:</b>',
+        'no_input'      => 1,
+        'is_html'       => 1,
+      },
+      {
+       'type'          => 'text',
+       'name'          => 'text',
+       'class'         => 'vep-input',
+      },
+      add_example_links(\@available_input_formats),
+      {
+        'type'          => 'div',
+        'element_class' => 'vep_left_input',
+        'inline'        => 1,
+        'children'      => [{
+          'node_name'   => 'span',
+          'class'       => '_ht ht',
+          'title'       => sprintf('File uploads are limited to %sMB in size.', $sd->ENSEMBL_TOOLS_CGI_POST_MAX->{'VEP'} / (1024 * 1024)),
+          'inner_HTML'  => '<b>Or upload file:</b>'
+        }]
+      },
+      {
+        'type'            => 'file',
+        'name'            => 'file',
+      }]
+  });
+
+  $input_fieldset->add_field({
+    'type'          => 'checklist',
+    'label'         => 'Results',
+    'field_class'   => [qw(_stt_yes _stt_allele)],
+    'values'        => [{
+      'name'          => "id",
+      'caption'       => $fd->{id}->{label},
+      'helptip'       => $fd->{id}->{helptip},
+      'value'         => 'yes',
+      'checked'       => 1
+    }, {
+      'name'          => "spdi",
+      'caption'       => $fd->{spdi}->{label},
+      'helptip'       => $fd->{spdi}->{helptip},
+      'value'         => 'yes',
+      'checked'       => 1
+    }, {
+      'name'          => "hgvsg",
+      'caption'       => $fd->{hgvsg}->{label},
+      'helptip'       => $fd->{hgvsg}->{helptip},
+      'value'         => 'yes',
+      'checked'       => 1
+    }, {
+      'name'          => "hgvsc",
+      'caption'       => $fd->{hgvsc}->{label},
+      'helptip'       => $fd->{hgvsc}->{helptip},
+      'value'         => 'yes',
+      'checked'       => 1
+    }, {
+      'name'          => "hgvsp",
+      'caption'       => $fd->{hgvsp}->{label},
+      'helptip'       => $fd->{hgvsp}->{helptip},
+      'value'         => 'yes',
+      'checked'       => 1
+    }, {
+      'name'          => "vcf_string",
+      'caption'       => $fd->{vcf_string}->{label},
+      'helptip'       => $fd->{vcf_string}->{helptip},
+      'value'         => 'yes',
+      'checked'       => 1
+    }]
+  }),
+
+  # Run button
+  $self->add_buttons_fieldset($form);
+
+  return $form;
+}
+
+sub add_example_links {
+  my $input_formats = shift;
+
+  if ($#$input_formats >= 0) {
+    return {
+      'type'    => 'noedit',
+      'noinput' => 1,
+      'is_html' => 1,
+      'caption' => sprintf('<span class="small"><b>Examples:&nbsp;</b>%s</span>',
+        join(', ', (map { sprintf('<a href="#" class="_example_input" rel="%s">%s</a>', $_->{'value'}, $_->{'caption'}) } @$input_formats ))
+      )
+    }
+  }
+  return;
+}
+sub get_non_cacheable_fields {
+  ## Abstract method implementation
+  return {};
+}
+
+sub js_panel {
+  ## @override
+  return 'VEPForm';
```

Comment:
Is this supposed to be VEPForm?

Reply:
Fixed to VRForm as well as tools/htdocs/components/20_VRForm.js L18